src/Symfony/Component/Debug/ErrorHandler.php

```diff
@@ -372,8 +370,17 @@ public function handleError($type, $message, $file, $line, array $context, array
         if (!$type || (!$log && !$throw)) {
             return $type && $log;
         }
+        $scope = $this->scopedErrors & $type;
 
-        if (isset($context['GLOBALS']) && ($this->scopedErrors & $type)) {
+        if (4 < $numArgs = func_num_args()) {
+            $context = $scope ? func_get_arg(4) : array();
+            $backtrace = 5 < $numArgs ? func_get_arg(5) : null;
```

Comment:
In that case, can you add a small comment? That way, we can remove this piece of code more easily if we decide to drop HHVM support one day.